ydb/core/tx/schemeshard/ut_login/ut_login.cpp

```diff
@@ -872,6 +872,52 @@ Y_UNIT_TEST_SUITE(TSchemeShardLoginTest) {
         });
         CheckUserIsLockedOutPermanently(runtime);
     }
+
+    Y_UNIT_TEST(CheckTimeOfUserCreating) {
+        TTestBasicRuntime runtime;
+        TTestEnv env(runtime);
+        ui64 txId = 100;
+
+        {
+            auto describe = DescribePath(runtime, TTestTxConfig::SchemeShard, "/MyRoot");
+            CheckSecurityState(describe, {.PublicKeysSize = 0, .SidsSize = 0});
+        }
+
+
+        {
+            std::chrono::time_point<std::chrono::system_clock> timeBeforCreateUser = std::chrono::system_clock::now();
+            CreateAlterLoginCreateUser(runtime, ++txId, "/MyRoot", "user1", "password1");
+            auto describeResult = DescribePath(runtime, TTestTxConfig::SchemeShard, "/MyRoot");
+            CheckSecurityState(describeResult, {.PublicKeysSize = 0, .SidsSize = 1});
+
+            NLogin::TLoginProvider tmpLoginProvider;
+            tmpLoginProvider.UpdateSecurityState(describeResult.GetPathDescription().GetDomainDescription().GetSecurityState());
+            const auto& sid = tmpLoginProvider.Sids["user1"];
+            UNIT_ASSERT(sid.CreatedAt > timeBeforCreateUser);
```

Comment:
Нужно еще снять еще один момент времени timeAflterCreateUser и добавить тут условие 
```suggestion
            UNIT_ASSERT(sid.CreatedAt > timeBeforCreateUser && sid.CreatedAt < timeAfterCreateUser);
```